pkgs/by-name/ob/objfw/package.nix

```diff
@@ -0,0 +1,46 @@
+{
+  autoconf,
+  autogen,
+  automake,
+  clang,
+  fetchFromGitHub,
+  lib,
+  stdenv,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "objfw";
+  version = "1.1.7";
+
+  src = fetchFromGitHub {
+    owner = "ObjFW";
+    repo = "ObjFW";
+    rev = "refs/tags/1.1.7-release";
+    hash = "sha256-0ylG/2ZSO3b8zdh6W9QJH5OJW9V344CCik1DduV5mhI=";
+  };
+
+  nativeBuildInputs = [
+    clang
+    automake
+    autogen
+    autoconf
+  ];
+
+  configurePhase = ''
+    runHook preConfigure
+    ./autogen.sh
+    ./configure --prefix=$out --without-tls
+    runHook postConfigure
+  '';
+
+  doCheck = true;
+  checkPhase = "make check";
```

Comment:
That's the default (or `test` depending on the conditional https://github.com/NixOS/nixpkgs/blob/d31617bedffa3e5fe067feba1c68b1a7f644cb4f/pkgs/stdenv/generic/setup.sh#L1485)

Reply:
Thanks for the review. I've integrated these changes now, and am awaiting OfBorg!